crates/consensus/consensus/src/lib.rs

```diff
@@ -434,6 +434,9 @@ pub enum ConsensusError {
         /// The block's timestamp.
         timestamp: u64,
     },
+    /// Other, likely an L2 error.
+    #[display("other, search logs for l2 error")]
```

Comment:
this message isn't super helpful, we shouldn't advise searching logs in an error,
could we make this a string instead?

Reply:
this is pre-req https://github.com/paradigmxyz/reth/issues/14499